kroxylicious-systemtests/src/main/java/io/kroxylicious/systemtests/Constants.java

```diff
@@ -159,9 +159,10 @@ public interface Constants {
      */
     String KAFKA_ADMIN_CLIENT_LABEL = "admin-client-cli";
     /**
-     * Image pull if not present.
+     * Image pull policy
      */
     String PULL_IMAGE_IF_NOT_PRESENT = "IfNotPresent";
+    String PULL_IMAGE_ALWAYS = "Always";
```

Comment:
nit: The diff reads wrong to me we now appear to be documenting `PULL_IMAGE_IF_NOT_PRESENT` as the current policy.
 
```suggestion
     * Image pull policies
     */
    String PULL_IMAGE_IF_NOT_PRESENT = "IfNotPresent";
    String PULL_IMAGE_ALWAYS = "Always";
```

Reply:
this was following a pattern in the constants file like:

```
/**
     * Restart policy
     */
    String RESTART_POLICY_ONFAILURE = "OnFailure";
    String RESTART_POLICY_NEVER = "Never";
```

but yeah maybe a pattern to change.

Reply:
Lets not keep repeating slightly  misleading patterns.